libs/utils-ts/src/VerifiedSale.ts

```diff
@@ -6,6 +6,9 @@ import { BuildVerifiedAction } from "./VerifiedAction";
 
 // We cannot use the following characters in the question/answer
 // Invalid characters: < or >, { or }, [ or ], %, &, #, \ or "
+export const validQuestion  = () => /^[^\<\>\{\}\[\]\%\&\#\\\"]{3,}$/g;
+export const validAnswer    = () => /^[^\<\>\{\}\[\]\%\&\#\\\"\s]{3,}$/g;
+export const validEmail     = () => /^[^@\s]+@[^@\s]+$/
 export const invalidAnswer   = () => /[\<\>\{\}\[\]\%\&\#\\\"\s]/g;
```

Comment:
I don't like duplicating the regex's here, but I still want the invalid regex so we can give clearer feedback to the user (validity is pass/fail in the above regexs)